src/L2VoteAggregator.sol

```diff
@@ -141,6 +141,6 @@ abstract contract L2VoteAggregator {
 
     // TODO: Check if this is inclusive
     return L1_BLOCK.number() <= internalVotingPeriodEnd(proposalId)
-      && L1_BLOCK.number() >= proposal.voteStart;
+      && L1_BLOCK.number() >= proposal.voteStart && !proposal.cancelled;
```

Comment:
super nit: might prefer the different condition statements each on their own line
maybe fmt has a problem with that though. feel free to ignore.

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/61/commits/a5a7ccdbfe2c0c1b6101e2f4eb8b97ab4d312d1b

Reply:
Ah it seems like this is due to the formatter, so we can't do the condition per line